src/codesearch.cc

```diff
@@ -206,18 +207,15 @@ class searcher {
 public:
     searcher(const code_searcher *cc,
              const query &q,
+             const intrusive_ptr<IndexKey> index,
```

Comment:
Sure! I opted for consistency despite being a bit puzzled by the naming, so I'm happy to make that change :)